static/styles/settings.css

```diff
@@ -952,9 +956,26 @@ input[type="checkbox"] {
         height: 22px;
         width: 22px;
     }
+}
 
-    .right {
-        float: right;
+.right_sidebar_style_values {
+    .preview {
+        background-color: inherit !important;
+        width: 95px;
```

Comment:
Where does 95px come from?

Reply:
I picked it after testing different values. If the width is less than this, the "Working remotely" status overflows to the next line.

Reply:
We just need to make the container bigger; I took care of this.